spec/text-editor-component-spec.coffee

```diff
@@ -568,10 +568,10 @@ describe "TextEditorComponent", ->
       nextAnimationFrame()
       expect(lineNumbersNode.style.backgroundColor).toBe 'rgb(255, 0, 0)'
 
-    it "hides or shows the gutter based on the '::isGutterVisible' property on the model and the global 'editor.showLineNumbers' config setting", ->
+    it "hides or shows the gutter based on the '::isDefaultGutterVisible' property on the model and the global 'editor.showLineNumbers' config setting", ->
```

Comment:
I'm thinking we should mention "line numbers" in our vocabulary for what you're calling the "default" gutter. Really it's carrying other purposes right now but I'd almost like to eventually move things like the fold indicators, bookmarks etc into their own gutter that sits to the right of the line numbers. Not sure on the name though since we'd have to use `are` instead of `is` for the plural... `::areLineNumbersVisible`, `::isLineNumbersGutterVisible`.